src/components/pager/pager.stories.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { State, Store } from '@sambego/storybook-state';
+import Pager from './pager';
+import notes from './notes.md';
+import OptionsHelper from '../../utils/helpers/options-helper/options-helper';
+
+const store = new Store({
+  currentPage: '1'
+});
+
+const handlePagination = (ev) => {
+  store.set({ currentPage: ev });
+};
+
+storiesOf('Pager', module)
+  .addParameters({
+    info: {
+      propTablesExclude: [State]
+    }
+  })
+  .add('default', () => {
+    const totalRecords = text('totalRecords', '100');
+    const pageSize = select('pageSize', OptionsHelper.pageSizes, Pager.defaultProps.pageSize);
+    const showPageSizeSelection = boolean('showPageSizeSelection', Pager.defaultProps.showPageSizeSelection);
+
+    return (
```

Comment:
In the current demo site:

![image](https://user-images.githubusercontent.com/26925206/53879498-9c6ecf80-4006-11e9-82db-745c700ca282.png)

This implementation:

![image](https://user-images.githubusercontent.com/26925206/53879474-819c5b00-4006-11e9-8087-c4c534075d41.png)

Whatever we put here it shouldn't be a string showing a call to `fromJS`.

Reply:
Fixed. There was no obvious / easy way to do this, so I had to resort to a custom TableComponent implementation, as per the example here https://github.com/storybooks/storybook/tree/master/addons/info#options-and-defaults